### DISCUSSION TITLE
I use nuxt3 and "@nuxtjs/i18n": "^8.0.0-beta.9"

### DISCUSSION THREAD

**Discussion by ibrahim-990** [nuxt-modules/i18n#1867] [General]
### Environment

test

### Reproduction

test

### Describe the bug

I get this warns in console: 
Component "i18n-t" has already been registered in target app.
Component "i18n-n" has already been registered in target app. 
"i18n-d" has already been registered in target app. 
Directive "t" has already been registered in target app.

how I can solve this warns please?

### Additional context

test

### Logs

```shell
test
```

**Comment by ineshbose** [comment]
This is not an issue and hence has been converted to a discussion, however, please provide more context in order to help us help you.

**Comment by ibrahim-990** [comment]
this my (nuxt.config.ts)
export default defineNuxtConfig({
  modules: [
    "@nuxtjs/i18n",
  ],
  i18n: {
    locales: ["en", "ar"],
    defaultLocale: "en",
  },
});
///////
and used like this in component
<p>
        {{ $t("site.our_services") }}
</p>
Just i do like this, and show that I attached warnings
